pyproject.toml

```diff
@@ -33,17 +34,18 @@ dependencies = [
   "bioio-base==1.0.6", 
   "dask[array]>=2021.4.1",
   "fsspec>=2022.8.0",
-  "imageio[ffmpeg]>=2.11.0,<2.28.0",
-  "ngff-zarr>=0.8.2,<0.9.1",
-  "numpy>=1.21.0,<2.0.0",
+  "imageio[ffmpeg]>=2.11.0",
+  "numpy>=1.21.0",
   "ome-types[lxml]>=0.4.0",
   "ome-zarr>=0.6.1",
+  # Use ngff-zarr for Python 3.10 or lower and ome-zarr-models for Python 3.11 or higher.
+  'ngff-zarr>=0.8.2,<0.9.1; python_version < "3.11"',
```

Comment:
Why less than 0.9.1 still - I thought that was a 3.9 issue?

Reply:
Oh I didn't try it really. the change here was mostly just the 'python_version < "3.11"''